technical-reports/format/composite-types.md

```diff
@@ -0,0 +1,220 @@
+# Composite types
+
+<p class="ednote" title="This section will be revisited">
+  Note: this proposal may not be in scope for the first version of the specification and requires further discussion.
+</p>
+
+The types described in the previous chapter are all for singular values (a color, a dimension, etc.). However, it can often be useful to group related values as a single token so that they can be used or referenced as a single unit. A typical example of this is a “typography style” as found in many design tools, which is a combination of the font name, weight, style, and color.
+
+AND: A combination of the above two examples. Mutually exclusive composites. (Grid styles)
```

Comment:
I think this line should be removed - it's part of a comment in the original Google doc, but doesn't make sense on its own like this.

```suggestion
```